server/api/src/rsModAVUMetadata.cpp

```diff
@@ -153,6 +158,10 @@ _rsModAVUMetadata( rsComm_t *rsComm, modAVUMetadataInp_t *modAVUMetadataInp ) {
                                    &modAVUMetadataInp->condInput);
     }
     else if ( strcmp( modAVUMetadataInp->arg0, "adda" ) == 0 ) {
+        const char* const msg = R"_("adda" is deprecated. Please use "add" with admin mode enabled instead.)_";
```

Comment:
suggestion...

"admin mode" -> "admin mode (-M)"

Reply:
are we fine with -M appearing in the response to clients that aren't icommands?

this question is the reason why i replaced ADMIN_KW with "admin mode".

Reply:
I like how the current wording is, personally.

Reply:
ah, right.   carry on.